rep-0151.rst

```diff
@@ -0,0 +1,260 @@
+REP: 151
+Title: Python 3 Support
+Author: Dirk Thomas
```

Comment:
Feel free to add yourself to the author list.

Reply:
Added in 43bd658